docker-compose.yaml

```
@@ -17,9 +17,11 @@ services:
     - DATABASE_TYPE=postgres
     - DATABASE_CLASS=mailman.database.postgresql.PostgreSQLDatabase
     - HYPERKITTY_API_KEY=someapikey
+    ports:
+    - "8001:8001" # API
```

One last comment, lets limit the host to loopback.
```suggestion
    - "127.0.0.0:8001:8001" # API
```